src/test/java/com/wikia/webdriver/common/core/api/ArticleContent.java

```diff
@@ -1,43 +1,57 @@
 package com.wikia.webdriver.common.core.api;
 
+import java.io.File;
+import java.util.ArrayList;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.message.BasicNameValuePair;
+
 import com.wikia.webdriver.common.contentpatterns.PageContent;
 import com.wikia.webdriver.common.core.TestContext;
 import com.wikia.webdriver.common.core.XMLReader;
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.core.url.UrlBuilder;
 
-import org.apache.commons.lang3.StringUtils;
-import org.apache.http.message.BasicNameValuePair;
-
-import java.io.File;
-import java.util.ArrayList;       
-
 public class ArticleContent extends ApiCall {
 
   private static String secret;
-  private static String baseURL;
-  private static ArrayList<BasicNameValuePair> PARAMS;
+  private String baseURL = new UrlBuilder().getUrlForWiki(Configuration.getWikiName())
+      + "/wikia.php?controller=Wikia\\Helios\\SampleController&method=edit&title=";
+  private ArrayList<BasicNameValuePair> PARAMS = new ArrayList<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this field "PARAMS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)